qopt_best_practices/swap_strategies/build_circuit.py

```diff
@@ -116,19 +116,27 @@ def apply_qaoa_layers(
 
 
 def create_qaoa_swap_circuit(
-    cost_operator: SparsePauliOp,
+    cost_operator: list[tuple[str, float]],
     swap_strategy: SwapStrategy,
     edge_coloring: dict = None,
     theta: list[float] = None,
     qaoa_layers: int = 1,
     initial_state: QuantumCircuit = None,
     mixer: QuantumCircuit = None,
 ):
-    """Create the circuit for QAOA.
-
-    Notes: This circuit construction for QAOA works for quadratic terms in `Z` and will be
-    extended to first-order terms in `Z`. Higher-orders are not supported.
-
+    """
+    Create the circuit for QAOA.
+    Notes: This circuit construction for QAOA works for quadratic terms in `Z` as well as
```

Comment:
```suggestion
    """Create the circuit for QAOA.
    
    Notes: This circuit construction for QAOA works for quadratic terms in `Z` as well as
```
Some IDE allow for a short comment followed by a long detailed comment. This is determined by the formating e.g.
```python
def foo():
    """A brief comment.
    
    A long comment that describes the functions
    and will typically span multiple lines.
    """
```